jakarta.ls/src/main/resources/transactions.json

```diff
@@ -1,3 +1,53 @@
 {
-
+	"Initialize a UserTransaction object via Injection": {
+		"prefix": "userTransactionInjection",
+		"body": [
+			"@Resource",
+			"UserTransaction ${1:TransactionName};",
```

Comment:
use camel case for the variables - `transactionName`, `functionName`, etc.

Reply:
Done!